.github/workflows/build-lint-test.yml

```diff
@@ -183,7 +183,7 @@ jobs:
       - backend
     environment:
       name: test
-      url: https://${{ github.sha }}.abacus-test.nl
+      url: https://${{ github.event.pull_request.head.sha || github.sha }}.abacus-test.nl
```

Comment:
I think we should always use `github.sha` because that's what we built. If we use `github.event.pull_request.head.sha`, we can have two builds with the same 'hash' that have been merged with different versions of `main` with no way to tell the difference.

Reply:
That is true - but I have no way of linking that virtual merge commit to a PR. GitHub does not expose this commit hash in the (graphql) API as far as I can see. For the GitHub state I prefer to have a single source of truth, thus "linking" these commits by an upload call would not be the nicest option.